packages/core/src/Account.ts

```diff
@@ -157,6 +167,7 @@ export class Account extends TypedEventEmitter<Events> {
     this.backendFeatures = this.apiClient.backendFeatures;
     this.cryptoProtocolConfig = cryptoProtocolConfig;
     this.nbPrekeys = nbPrekeys;
+    this.enableMLS = () => this.backendFeatures.supportsMLS && !!this.cryptoProtocolConfig?.mls;
```

Comment:
Let's maybe rename it to `isMLSEnabled`, `enableMLS` sounds like it will enable mls in the core